### PR TITLE
Inverting y axis in accelerometer

### PR DESCRIPTION
**Pull request by riknoll**
Fixes https://github.com/Microsoft/pxt-adafruit/issues/304

The Y axis of the accelerometer doesn't match the markings on the circuit playground or the simulator behavior. This PR inverts that value and also fixes the simulator tilt up and down events so that they match the hardware.

### CODE REVIEW

**Comment by pelikhan** [comment]
This should be constant that can be overriden per target. It might not be the case for other boards.

**Comment by riknoll** [comment]
Sure. I guess the best way to do that is add a constants.cpp file that can be overridden?

**Comment by riknoll** [comment]
@pelikhan fixed

File: libs/accelerometer/accelerometer.cpp [hunk 1]
```diff
@@ -103,13 +106,46 @@ enum class Gesture {
...
+public:
+    InvertableLIS3DH(codal::mbed::I2C &_i2c, Pin &_int1, uint16_t address = LIS3DH_DEFAULT_ADDR, uint16_t id = DEVICE_ID_ACCELEROMETER, CoordinateSystem coordinateSystem = SIMPLE_CARTESIAN) : LIS3DH(_i2c, _int1, address, id, coordinateSystem) {}
+    int getX();
+    int getY();
+    int getZ();
+};
+
+int InvertableLIS3DH::getX() {
+#if INVERT_ACC_X_AXIS
+    return LIS3DH::getX() * -1;
```
Comment (pelikhan) [inline libs/accelerometer/accelerometer.cpp]: ``- LIS3DH::getX()`` ?

**Comment by riknoll** [comment]
@pelikhan yup, tested. With https://github.com/Microsoft/pxt-adafruit/pull/305 the simulator and hardware now match.

**Comment by riknoll** [comment]
BTW, this is a breaking change. I'll merge it and do the required releases tomorrow

**Pull request merged**